RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -663,6 +663,12 @@ void finalizeMaximize(boolean speculative)
 		//100 to 200 bonus for diminishing returns when drams already high
 		addBonusToMaximize($item[tiny stillsuit], (100 + to_int(100*min(1,(10.0 / max(1,auto_expectedStillsuitAdvs()))))));
 	}
+	if(my_familiar() == $familiar[Jill-of-All-Trades] && item_amount($item[LED candle]) > 0)
```

Comment:
I would do this at the end of `preAdvUpdateFamiliar()` since it's familiar specific. The tiny stillsuit & crystall ball are being handled here because they're generic and apply to all familiars.

Reply:
good call. Originally I had a bonus to maximizer here, but decided later it was fine to simply appropriately configure it. Will fix